File: DSA/array/maxSunArray/brute.cpp
#include <vector>  
#include <algorithm>
#include <limits>
#include <iostream>


class Solution { 
public :
    int maxSubArray(std::vector<int>& nums){

        int n = nums.size();
        int maxSum = std::numeric_limits<int>::min();
        
        for(int i = 0; i < n; i++){
            int sum = 0;
            for(int j = i; j < n; j++){
                sum+= nums[j];
                maxSum = std::max(maxSum, sum);
            }
        }

        return maxSum;

    }

    int main() {

        std::vector<int> nums = {-2, 1, -3, 4, -1, 2, 1, -5, 4};
        int maxSum = maxSubArray(nums);
        std::cout << "Answer: " << maxSum;
        return 0;
    }
};

File: 1 TwoSum/brute.py
from typing import List


class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        
        n = len(nums)
        for i in range(n-1):
            for j in range(i+1, n):
                sum = nums[i] + nums[j]
                
                if sum == target:
                    return [i,j]
            